docker/Dockerfile

```diff
@@ -0,0 +1,25 @@
+
+FROM openjdk:11.0.2-jre-slim-stretch
+
+COPY accountgenerator /opt/accountgenerator/
+WORKDIR /opt/accountgenerator
+
+# Expose services ports
+# 7545 HTTP JSON-RPC
+EXPOSE 7545
+
+ENTRYPOINT ["/opt/accountgenerator/bin/accountgenerator"]
+
+# Build-time metadata as defined at http://label-schema.org
+ARG BUILD_DATE
+ARG VCS_REF
+ARG VERSION
+LABEL org.label-schema.build-date=$BUILD_DATE \
+      org.label-schema.name="accountgenerator" \
+      org.label-schema.description="Ethereum account generator application" \
+      org.label-schema.url="https://docs.accountgenerator.pegasys.tech/" \
```

Comment:
is this `pegasys.tech` ? should it be an adhara url instead?

Reply:
The tool was created solely to generate accounts to be used with Pegasys EthSigner. Most of its non-crypto code was taken from EthSigner to be able to function exactly like the EthSigner JSON RPC server. I can remove all reference to Pegasys if that is best?